src/components/Navigator/NavigatorCard.vue

```diff
@@ -472,6 +476,22 @@ export default {
         this.$refs.scroller.scrollToItem(index);
       }
     },
+    // TODO: This is very much a test
+    handleBlur(event) {
+      // if there is a related target, do nothing
+      if (event.relatedTarget !== null) return;
+      // if there is no related target re-focus the item
+      console.log('lost focus', event.target);
```

Comment:
We should remove these debugging `console.log` statements before merging this in a PR.